app/controllers/api/scores_controller.rb

```diff
@@ -5,7 +5,7 @@ class ScoresController < ApplicationController
     before_action :validate_score_user_id, only: :destroy
 
     def user_feed
-      scores = Score.all.order(played_at: :desc, id: :desc)
+      scores = Score.all.includes(:user).order(played_at: :desc, id: :desc)
```

Comment:
We should not make the same changes in two pull requests. This is already done in https://github.com/SuciuIoanaMagdalena/golfr_backend/pull/1.
If we include it here too, we'll get into git conflicts.